router-tests/prometheus_test.go

```diff
@@ -50,8 +50,6 @@ func TestPrometheus(t *testing.T) {
 			requestTotalMetrics := requestTotal.GetMetric()
 
 			require.Len(t, requestTotalMetrics, 2)
-			require.Len(t, requestTotalMetrics[0].Label, 12)
```

Comment:
Why did you remove it?

Reply:
We check the entire contents immediately after, so this just produces unhelpful test failures

Reply:
Alright, thanks